src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -57,12 +57,19 @@ const Body = BaseForm.extend(Object.assign(
         : loc('mfa.calling', 'login');
       const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
+      let nicknameText = '';
+      if (this.model.get('nickname')) {
+        nicknameText = '(' + this.model.escape('nickname') + ')';
+      }
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
```

Comment:
Similar to [this](https://github.com/okta/okta-signin-widget/pull/3342/files#diff-613a9a51a022846714b15e19706db7688f92eab855376f7e72de20bd5cd8c431R42-R48) comment,will want to create separate translation keys for this as well.